pkg/plugins/backendplugin/client.go

```diff
@@ -30,9 +30,12 @@ var handshake = goplugin.HandshakeConfig{
 	MagicCookieValue: grpcplugin.MagicCookieValue,
 }
 
-func newClientConfig(executablePath string, logger log.Logger, versionedPlugins map[int]goplugin.PluginSet) *goplugin.ClientConfig {
+func newClientConfig(executablePath string, env []string, logger log.Logger, versionedPlugins map[int]goplugin.PluginSet) *goplugin.ClientConfig {
+	cmd := exec.Command(executablePath)
+	cmd.Env = env
```

Comment:
I thin we might want to merge `env` with the OS environment. Otherwise, the plugin's environment will only be constituted of the configuration parameters, and lose all other environment parameters (f.ex. $PATH). Is this intentional?  It's an easy consequence to forget when setting a process' environment variables.

Reply:
That's automatically added by hashicorp/go-plugin: https://github.com/hashicorp/go-plugin/blob/991a34ba3526019b82938e1c292e1fe4b5782f43/client.go#L516-L525

Reply:
@marefr Oh that's great.